web/libs/editor/src/lib/AudioUltra/Visual/Visualizer.ts

```diff
@@ -253,7 +253,7 @@ export class Visualizer extends Events<VisualizerEvents> {
     }
 
     this.invoke("initialized", [this]);
-    this.transferImage();
+    setTimeout(() => this.draw(), 10);
```

Comment:
Instead of binding to a timeout of 10ms, why not just leave it to the next animation frame?
```suggestion
    requestAnimationFrame(() => this.draw());
```

Reply:
yes absolutely! great idea!